{"uuid": "9c03acab-c97d-45ea-96b4-4325a0ad4187", "children": ["b1bcd9b1-32da-47a8-95fc-4c042ae0e4b8"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5650a690affa <unknown>\n#1 0x5650a63c9970 <unknown>\n#2 0x5650a641b385 <unknown>\n#3 0x5650a641b5b1 <unknown>\n#4 0x5650a646a3c4 <unknown>\n#5 0x5650a64412bd <unknown>\n#6 0x5650a646770c <unknown>\n#7 0x5650a6441063 <unknown>\n#8 0x5650a640d328 <unknown>\n#9 0x5650a640e491 <unknown>\n#10 0x5650a68d242b <unknown>\n#11 0x5650a68d62ec <unknown>\n#12 0x5650a68b9a22 <unknown>\n#13 0x5650a68d6e64 <unknown>\n#14 0x5650a689dbef <unknown>\n#15 0x5650a68f9558 <unknown>\n#16 0x5650a68f9736 <unknown>\n#17 0x5650a6909e76 <unknown>\n#18 0x7f8420c9caa4 <unknown>\n#19 0x7f8420d29c3c <unknown>\n\n", "trace": "  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/src/tests/ui_tests/conftest.py\", line 30, in driver\n    WebDriverWait(driver, 10).until(\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1743000359297, "stop": 1743000370779}], "start": 1743000359297, "stop": 1743000370786}